### DISCUSSION TITLE
HJT: Main discussion thread - improvement & development & news

### DISCUSSION THREAD

**Discussion by dragokas** [dragokas/hijackthis#137] [General]
Hi, everyone !!!

**Welcome to the HJT development thread.**

Here you can ask fast basic questions about HiJackThis Fork.
However, for our convenience, it's better that you [create new issue](https://github.com/dragokas/hijackthis/issues/new) for your case, especially for bug reports.

**Don't forget** to read [HJT tutorial](https://github.com/dragokas/hijackthis/wiki/HJT:-Tutorial) and [Wiki-pages](https://github.com/dragokas/hijackthis/wiki) before asking questions.
**Do not post HJT logs here** for analyzing. Instead, create a new thread (issue).

Most of the news is published most quickly on general forum in our [Russian-speaking community](https://safezone.cc/threads/hijackthis-fork-i-voprosy-k-razrabotchikam.28770/).
This topic will be updated with news as far as possible.

To post here you need to process a [simple registration](https://github.com/join) on GitHub.

Sometimes, new version of binary coming before pushing the source code. If you want you can test actual alpha/beta-version by this link: https://dragokas.com/tools/HiJackThis_test.zip

I wish you nice day and no viruses,
Stanislav.

**Comment by colok** [comment]
I introduce myself: I'm Colok, the administrator of the website www.Colok-Traductions.com where I've been offering french free translations for more than 18 years

![sans titre](https://user-images.githubusercontent.com/6083274/48409012-0c490980-e73b-11e8-9d6f-9ea5bc69e76b.jpg)

Before Merijn, the author of HiJackThis, sold it to Micro Trend, I made the French translation of this program. If you are interested, I can offer you this french translation for free.

You can contact me at this address xxxxxxxxxx

You can see the last French translation before purchasing the program by Trend Micro : https://www.colok-traductions.com/modules.php?name=Blogs&op=billet&bid=163#billet

Best regards
Colok

**Comment by dragokas** [comment]
Hi, colok!
Thank you. I will contact with you by email.

**Comment by dragokas** [comment]
Unfortunately I can't send you an email:
> Remote Server returned '550 5.5.0 Requested action not taken: mailbox unavailable.'

That's why I will double my message here:

I'm thankful for your offer to give me French translation of original HiJackThis.

However, I am afraid that I will have to put it in the archive for a long time, because I have no specialists to continue translation based on your version.

Since original HJT, my fork contains about 1500 lines of text (when original ~ 250 lines).

If you can offer to make a complete translation, or you know the people who can translate it free of charge, it would be very interesting for us.

Also, when I open your website, I'm constantly see this warning instead of real webpage (see screenshot).

![111](https://user-images.githubusercontent.com/19956568/48427546-a62ba900-e771-11e8-9dc6-1a519cb87e06.png)

Have a nice day,
Stanislav Polshyn.

**Comment by colok** [comment]
Hi,

I can do it

Best regards
Colok


________________________________
De : Stanislav Polshyn <notifications@github.com>
Envoyé : mardi 13 novembre 2018 17:26
À : dragokas/hijackthis
Cc : colok; Comment
Objet : Re: [dragokas/hijackthis] HJT: Main discussion thread - improvement & development & news (#4)


Unfortunately I can't send you an email:

Remote Server returned '550 5.5.0 Requested action not taken: mailbox unavailable.'

That's why I will double my message here:

I'm thankful for your offer to give me French translation of original HiJackThis.

However, I am afraid that I will have to put it in the archive for a long time, because I have no specialists to continue translation based on your version.

Since original HJT, my fork contains about 1500 lines of text (when original ~ 250 lines).

If you can offer to make a complete translation, or you know the people who can translate it free of charge, it would be very interesting for us.

Also, when I open your website, I'm constantly see this warning instead of real webpage (see screenshot).

[111]<https://user-images.githubusercontent.com/19956568/48427546-a62ba900-e771-11e8-9dc6-1a519cb87e06.png>

Have a nice day,
Stanislav Polshyn.

**Comment by colok** [comment]
For this,


[111]<https://user-images.githubusercontent.com/19956568/48427546-a62ba900-e771-11e8-9dc6-1a519cb87e06.png>

See with my webmaster !

Or contact him xxxxxxxxxxx

Best regards
Colok

________________________________
De : Stanislav Polshyn <notifications@github.com>
Envoyé : mardi 13 novembre 2018 17:26
À : dragokas/hijackthis
Cc : colok; Comment
Objet : Re: [dragokas/hijackthis] HJT: Main discussion thread - improvement & development & news (#4)


Unfortunately I can't send you an email:

Remote Server returned '550 5.5.0 Requested action not taken: mailbox unavailable.'

That's why I will double my message here:

I'm thankful for your offer to give me French translation of original HiJackThis.

However, I am afraid that I will have to put it in the archive for a long time, because I have no specialists to continue translation based on your version.

Since original HJT, my fork contains about 1500 lines of text (when original ~ 250 lines).

If you can offer to make a complete translation, or you know the people who can translate it free of charge, it would be very interesting for us.

Also, when I open your website, I'm constantly see this warning instead of real webpage (see screenshot).

[111]<https://user-images.githubusercontent.com/19956568/48427546-a62ba900-e771-11e8-9dc6-1a519cb87e06.png>

Have a nice day,
Stanislav Polshyn.

**Comment by colok** [comment]
It's better to send me email with a real address, not via this forum...

**Comment by dragokas** [comment]
> I can do it

What you mean? Can you make a translation? It would be excellent.

> It's better to send me email with a real address, not via this forum...

I can't send it to you (I tried my 2 different servers). Your email system refuse my emails.
> Remote Server returned '550 5.5.0 Requested action not taken: mailbox unavailable.'

Here is my email: xxxxx

**Comment by colok** [comment]
I sent you a message through another email

Regards
Colok


________________________________
De : Stanislav Polshyn <notifications@github.com>
Envoyé : mardi 13 novembre 2018 22:07
À : dragokas/hijackthis
Cc : colok; Comment
Objet : Re: [dragokas/hijackthis] HJT: Main discussion thread - improvement & development & news (#4)


I can do it
What you mean? Can you make a translation? It would be excellent.

It's better to send me email with a real address, not via this forum...
I can't send it to you (I tried my 2 different servers). Your email system refuse my emails.
Remote Server returned '550 5.5.0 Requested action not taken: mailbox unavailable.'

Here is my email: dragokas safezone.cc

**Comment by dragokas** [comment]
French translation is done in v2.9.0.11.
Well done, Colok!
Very fast and qualitatively.

Final corrections will come soon.

**Comment by colok** [comment]
Great !

**Comment by peddy22** [comment]
Where can I post my log to be analysed? Me and a couple of other people from Reddit are having some issues with a ebay hijacker

**Comment by dragokas** [comment]
Hi,

each one have to create separate topic in this section: https://github.com/dragokas/hijackthis/issues

by following this instruction: 
  * [How to make a request for help in the PC cure section](https://github.com/dragokas/hijackthis/wiki/How-to-make-a-request-for-help-in-the-PC-cure-section%3F)

**Comment by mauriciogracia** [comment]
First of all thanks for keeping this project alive and for your improvements

Would it be possible to provide a FILTER on the found results of HijackThis ? 

When SEARCH is executed the filter will only shows lines that match that pattern, this combined with the select all current results will allow you to FIX all the results that say "file missing" for example or when you are looking for a specific process or file name.

Here is a very crude prototype

![image](https://user-images.githubusercontent.com/2321661/97818444-7896dd80-1c70-11eb-8520-ce066d06902c.png)

**Comment by dragokas** [comment]
Hi. Understand. It is possible. Can you open a new issue for it?

**Comment by mauriciogracia** [comment]
Great, here is the issue - https://github.com/dragokas/hijackthis/issues/114

**Comment by gaucho1978** [comment]
An unexpected error has occurred !

Please help us improve HiJackThis by reporting this error.

Error message has been copied to clipboard.
Click 'Yes' to submit.

Details:
Error at function: GetMultiSZ
Error # 5 - Invalid procedure call or argument
HRESULT: 
LastDllError # 0
Trace info: hHive: -2147483646 System\CurrentControlSet\Services\aswbIDSAgent\DependOnService bUseWow64: False 

Windows version: x64 Windows 10 (Home), 10.0.17134.1304, Service Pack: 0
HiJackThis Fork by Alex Dragokas v.2.10.0.17
--- EOF ---

**Reply by dragokas** [reply]
@gaucho1978 thank you for let us know.
Please, open new topic about this case:
https://github.com/dragokas/hijackthis/issues/new?assignees=&labels=bug&template=bug_report.md&title=

**Comment by ashrea06** [comment]
Hey everyone upon logging in, i attempted to scan the system, got this error message :

![image](https://user-images.githubusercontent.com/77737581/222240614-c4421ba9-8441-47c1-ac26-a8f47c21710a.png)



Can I please get some help with regards to this. 

Thank you.

**Reply by dragokas** [reply]
@Yashil06 thank you for report. I converted your request into the new issue: https://github.com/dragokas/hijackthis/issues/208
Please, continue all the next conversation there.

**Comment by ashrea06** [comment]
Thank you.

**Reply by ashrea06** [reply]
Kindly close this, as this has been resolved. Thank you sir.

**Comment by anishre** [comment]
Bit late to the party but I got this when doing my first scan:
![image](https://github.com/user-attachments/assets/1b245b69-f286-4a1e-b58d-184af1d88c9e)

**Reply by dragokas** [reply]
Hi, thanks for the report. Please, use "Issues" section next time.
This issue is now tracked here: https://github.com/dragokas/hijackthis/issues
